src/com/hms/flexyosisoftconnector/DataPoint.java

```diff
@@ -16,8 +16,19 @@
    public static final byte TYPE_INT     = 2;

```

Comment:
There is a lot of changes in this commit and the commit title is very vague. Ideally a commit like this will have a significant commit message to go along with the title to describe the changes implemented in the commit.